library/src/webapp/content/gateway/features.html

```diff
@@ -231,7 +231,7 @@ <h4 class="modal-title">Communication and Collaboration</h4>
 					<strong>Contact Us</strong> - Quickly and easily direct questions and feedback to the correct source. With one click, the Contact Us tool can send an email to an institution’s helpdesk, with all relevant info automatically included.<br><br>
 					<strong>Email</strong> - Send mail to participants in your site using the institution’s official email system.<br><br>
 					<strong>Email Archive</strong> - Create a mailing list for your course or worksite, allowing for many-to-many broadcasting; access and search an archive of email sent to this address.<br><br>
-					<strong>Forums</strong> - Create, moderate and manage discussion topics and groups within a course and send private messages to participants; options to grade discussions, set word count, permissions, etc.<br><br>
+					<strong>Discussions</strong> - Create, moderate and manage discussion topics and groups within a course and send private messages to participants; options to grade discussions, set word count, permissions, etc.<br><br>
```

Comment:
Please move Discussions up between Contact Us and Email to keep the list alphabetized. Thanks.

Reply:
Done. Thanks for the review!